runbotics/runbotics-orchestrator-ui/src/src-app/utils/axios.ts

```diff
@@ -28,7 +28,7 @@ axiosInstance.interceptors.response.use(
 const axiosApi = axios.create();
 
 axiosApi.interceptors.response.use((res) => res, (err) => {
-    if (err.request.status === 401 && !err.request.responseURL.includes('/api/authenticate')) {
+    if (err.request.status === 401 && !err.request.responseURL.includes('/api/scheduler/auth/authenticate')) {
```

Comment:
shouldn't it stay as we discussed? /api/authenticate should be now proxied to scheduler